health/health_log.c

```diff
@@ -486,6 +517,18 @@ inline ALARM_ENTRY* health_create_alarm_entry(
     ae->old_value_string = strdupz(format_value_and_unit(value_string, 100, ae->old_value, ae->units, -1));
     ae->new_value_string = strdupz(format_value_and_unit(value_string, 100, ae->new_value, ae->units, -1));
 
+    if (likely(info)) {
+        char *m;
+        ae->info = strdupz(info);
+
+        while ( m = strstr (ae->info, "$family") ) {
```

Comment:
The unique difference between this loop and the loop inside `health_json` is an argument for `find_and_replace`, is necessary to have an output different for both loops? If it is not, please, create an unique function for both loops.